sphinx/source/docs/user_guide/server.rst

```diff
@@ -453,101 +458,90 @@ plot):
     N = 200
 
 .. warning::
-  The request object is provided so that values such as ``arguments`` may be
-  easily inspected. Calling any of the Tornado methods such as ``finish()`` or
-  writing directly to ``request.connection`` is unsupported and will result in
-  undefined behavior.
+  The request object makes inspecting values, such as ``arguments``, easy.
+  However, calling any of the Tornado methods, such as ``finish()``,  or
+  writing directly to ``request.connection`` results in undefined behavior.
```

Comment:
I'd like to keep the word "unsupported" in there somewhere to make the discouragement as strong and explicit as possible.